version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.11.0"
+version in ThisBuild := "0.11.0 -SNAPSHOT"
```

Comment:
We probably should actually swap all of them to work like this. We've seen some funky errors and mixed classpath's if you have a published version in the version.sbt. Not sure if its a new sbt thing or what. But can be a different PR